.eslintrc.json

```diff
@@ -8,13 +8,9 @@
     {
       "files": ["*.md"],
       "plugins": ["markdown"],
-      "env": {"es6": true},
       "globals": {"checkPermission": false, "findRoles": false},
       "rules": {
-        "strict": ["off"],
-        "no-extra-semi": ["off"],
-        "semi-style": ["off"],
-        "semi": ["off"]
+        "no-unused-vars": ["error", {"varsIgnorePattern": "^(checkPermission|findRoles)$"}]
```

Comment:
I prefer this to `<!-- eslint-disable no-unused-vars -->`, but I'm happy to revert this change if you have a different preference.